spec/lib/vcr/errors_spec.rb

```diff
@@ -72,12 +72,20 @@ def request_with(values)
           end
         end
 
-        it 'mentions the details about the current casette' do
+        it 'mentions the details about the single cassette when there is one cassette' do
           VCR.use_cassette('example') do
             expect(message).to match(/VCR is currently using the following cassette:.+example.yml/m)
           end
         end
 
+        it 'mentions the details about all cassettes when there are a few cassettes' do
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.